runtime/doc/lua.txt

```diff
@@ -1811,21 +1811,22 @@ set({mode}, {lhs}, {rhs}, {opts})                           *vim.keymap.set()*
                    -- You can map a key for specific buffer
                    vim.keymap.set('n', '<leader>w', "<cmd>w<cr>", { silent = true, buffer = 5 })
 
-                   -- Also expr keymaps too. Notice the replace_termcodes when making Lua expr maps.
+                   -- Expr mappings
                    vim.keymap.set('i', '<Tab>', function()
                      return vim.fn.pumvisible() == 1 and "<C-n>" or "<Tab>"
-                   end, { silent = true, expr = true, replace_termcodes = true })
-                   -- While setting <Plug> mapping remember to set remap to true
-                   vim.keymap.set('n', '[', '<Plug>(MatchitNormalMultiBackward)', {silent = true, remap = true })
+                   end, { silent = true, expr = true })
+                   -- <Plug> mappings
+                   vim.keymap.set('n', '[', '<Plug>(MatchitNormalMultiBackward)', {silent = true})
 <
 
                 A pit fall of this is if you want to make keymap like: >
 
                     vim.keymap.set('n', 'asdf', require('jkl').my_fun)
 <
 
-                `require('jkl)` get's evaluated durimg this call. If you want to avoid this
-                cost at startup you can wrap it in a function, for example: >
+                require('jkl') gets evaluated durimg this call. If you want to
```

Comment:
```suggestion
                require('jkl') gets evaluated during this call. If you want to
```

Reply:
Wait what I just fixed that 🤷‍♂️

Reply:
It's already been fixed . You're looking at older diff.